File: gpt_multiagent_python_openai_jira_social_plus/api/health.py
from http.server import BaseHTTPRequestHandler
import json
from datetime import datetime

class handler(BaseHTTPRequestHandler):
    def do_GET(self):
        self.send_response(200)
        self.send_header('Content-Type', 'application/json')
        self.send_header('Access-Control-Allow-Origin', '*')
        self.end_headers()
        
        response = {
            "status": "healthy",
            "service": "kaspa-community-tool",
            "platform": "vercel",
            "timestamp": datetime.utcnow().isoformat() + "Z",
            "version": "0.3.0",
            "uptime": "operational"
        }
        
        self.wfile.write(json.dumps(response).encode('utf-8'))

File: gpt_multiagent_python_openai_jira_social_plus/api/health.js
export default function handler(req, res) {
  res.setHeader('Access-Control-Allow-Origin', '*');
  res.setHeader('Content-Type', 'application/json');

  const response = {
    status: "healthy",
    service: "kaspa-community-tool",
    platform: "vercel",
    runtime: "node.js",
    timestamp: new Date().toISOString(),
    version: "0.4.0"
  };

  res.status(200).json(response);
}